src/System.Net.HttpListener/tests/System.Net.HttpListener.Tests.csproj

```diff
@@ -34,5 +34,8 @@
     </Compile>
     <Compile Include="HttpListenerTimeoutManagerTests.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsOSX)'=='true'">
+    <TestCommandLines Include="ulimit -n 70000" />
```

Comment:
When I tried this on Tomas' machine it told me 70000 was invalid.  But it's worth trying, I guess.